pdns/pdnsutil.cc

```diff
@@ -370,6 +370,12 @@ static int checkZone(DNSSECKeeper &dk, UeberBackend &B, const DNSName& zone, con
         cout<<"[Warning] SOA autocomplete is deprecated, missing field(s) in SOA content: "<<rr.qname<<" IN " <<rr.qtype.toString()<< " '" << rr.content<<"'"<<endl;
       }
 
+      if(parts.size() >= 2) {
+        if(parts[1].find('@') != string::npos) {
+          cout<<"[Warning] Found @-sign in SOA RNAME, should probably be a dot (.): "<<rr.qname<<" IN " <<rr.qtype.toString()<< " '" << rr.content<<"'"<<endl;
```

Comment:
```suggestion
          cout<<"[Warning] Found @-sign in SOA RNAME, should probably be a dot (.): "<<rr.qname<<" IN " <<rr.qtype.toString()<< " '" << rr.content<<"'"<<endl;
          numwarnings++;
```

Reply:
Somehow you managed to suggest a change that changes nothing. Impressive. Looks like a GitHub bug :-)

Reply:
oh, you added a line.